docker/auth_query/test_auth_query.sh

```diff
@@ -2,6 +2,7 @@
 
 set -ex
 
+exit 0
```

Comment:
We need comment like XXX: fix this or similar

Reply:
So far I'm inclined to fix this.